gateway/src/apicast/policy/keycloak_role_check/keycloak_role_check.lua

```diff
@@ -87,7 +89,12 @@ local function build_templates(scopes)
     scope.resource_template_string = create_template(
       scope.resource, scope.resource_type)
 
+    if not scope.methods then
```

Comment:
Can we change the name of this method? It's not longer about building templates only.

Reply:
What if `scope.methods` is an empty array? I think we should also set `scope.methods` to `{ any_method }`.